src/pages/Withdrawals/index.tsx

```diff
@@ -892,7 +908,10 @@ export const Withdrawals = () => {
                 </ul>
               </li>
               <li>
-                <FormattedMessage defaultMessage="Is the effective balance maxed out at 32?" />
+                <FormattedMessage
+                  defaultMessage="Is the effective balance maxed out at {PRICE_PER_VALIDATOR}?"
```

Comment:
Yeah good call, might as well add the ticker to these as well... I will adjust this locally since committing just this will change the translation hash again; will just need to update the existing translated strings to have that new hash as a key.